frontend/.gitignore

```diff
@@ -17,7 +16,8 @@
 .env.development.local
 .env.test.local
 .env.production.local
-
+.idea
+package-lock.json
```

Comment:
I am not an expert with node but isn't it good to commit this file so that everyone has the same versions of the dependencies?

Reply:
Agree! Reverting this